ui/components/component-library/avatar-token/avatar-token.js

```diff
@@ -27,52 +29,44 @@ export const AvatarToken = ({
     setShowFallback(!tokenImageUrl);
   }, [tokenImageUrl]);
 
-  const baseStyle = showHalo ? { position: `relative` } : {};
-  const sizePercentage = showHalo ? '62.5%' : '100%';
-
-  const style = size
-    ? {
-        height: sizePercentage,
-        width: sizePercentage,
-        borderRadius: '50%',
-        zIndex: '2',
-      }
-    : {};
-
   const handleOnError = () => {
     setShowFallback(true);
   };
 
   return (
     <BaseAvatar
       size={size}
-      style={baseStyle}
       display={DISPLAY.FLEX}
       alignItems={ALIGN_ITEMS.CENTER}
       justifyContent={JUSTIFY_CONTENT.CENTER}
       className={classnames(
-        'base-avatar',
-        `base-avatar--size-${size} avatar-token`,
+        'avatar-token',
+        showHalo && 'avatar-token--with-halo',
+        className,
       )}
       {...{ backgroundColor, borderColor, ...props }}
     >
       {showFallback ? (
-        tokenName?.[0]?.toUpperCase() ?? '?'
+        tokenName?.[0] ?? '?'
       ) : (
```

Comment:
@georgewrmarshall shouldn't we keep `tokenName?.[0]?.toUpperCase()` to keep the initial uppercase?

Reply:
https://github.com/MetaMask/metamask-extension/pull/15575/files#r945938185